tasks.py

```diff
@@ -265,12 +275,12 @@ def load_addresses(ctx, files=[]):
     if ctx.addresses.get("bano", {}).get("file"):
         options.append("--bano")
         options.append(ctx.addresses.bano.file)
-    if ctx.addresses.get("oa", {}).get("file"):
+    if ctx.addresses.get("oa", {}).get("path"):
         options.append("--openaddresses")
-        options.append(ctx.addresses.oa.file)
-    if ctx.get("osm", {}).get("file"):
+        options.append(ctx.addresses.oa.path)
```

Comment:
`addresses.oa.file` is still mentionned in the `invoke.yaml` file that is used as a documentation for config values.  
It should be replaced in there too, right ?

Reply:
That's correct! It also means I should document configuration for `addresses.osm.*`